require ubuntu 20.04, test on debian 11, require Python 3.8 (#856)
jupyterhub/the-littlest-jupyterhub: docs/index.rst

Installation
============

The Littlest JupyterHub (TLJH) can run on any server that is running **Debian 11** or **Ubuntu 20.04** or **22.04** on an amd64 or arm64 CPU architecture.
We aim to support 'stable' and Long-Term Support (LTS) versions.

> Newer versions are likely to work with little or no adjustment, but these are the officially supported and tested versions.

Reviewer: ```suggestion
Newer versions are likely to work with little or no adjustment, but these are not officially supported or tested.
```

Installation
============

> The Littlest JupyterHub (TLJH) can run on any server that is running **Debian 10** or **Ubuntu 20.04** on a amd64 or arm64 CPU architecture. Earlier versions of Ubuntu and Debian are not supported, as are other Linux distributions.

Reviewer 2: We test with Ubuntu 22.04 as well, so that makes sense to include still, and I figure we look to support Debian 11 as well.

Author: Yup, sometimes it says `>=`, but here it implies exact. What do we want to claim? Do we want to claim only exactly `20.04, 22.04`, or `>=20.04`?

Reviewer 2: I figure that we aim to support LTS releases of Ubuntu 20.04 and Debian 11 or higher, and currently we test for Ubuntu 20.04, 22.04, and debian 11.

Author: added:

> We aim to support 'stable' and Long-Term Support (LTS) versions.
Newer versions are likely to work with little or no adjustment, but these are the officially supported and tested versions.

to try to communicate that 24.04 _may_ work, but if it's not in the list above, it's not 'supported'

Reviewer 2: Looked at the commit, I think it was great!!
